components/centraldashboard/public/components/main-page.js

```diff
@@ -227,8 +227,8 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
     _onPlatformInfoResponse(responseEvent) {
         const {response} = responseEvent.detail;
         this.platformInfo = response;
-        if (this.platformInfo.kubeflowVersion) {
-            this.buildVersion = this.platformInfo.kubeflowVersion;
+        if (response.kubeflowVersion) {
```

Comment:
Any reason for the change here?

Reply:
Brevity; can revert if preferred.

Reply:
I don't know of a guideline that applies but it seems like we should try to avoid non-semantic changes unless part of an intentional refactoring effort.